src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
It can be also optimized by moving the creation of the `Random` object outside the method. Creating a new Random object every time the method is called may lead to less random behavior due to the default seed being based on the current time